### PR TITLE
AWS ALB Ingress Controller renamed to AWS Load Balancer Controller

### PR DESCRIPTION
**Pull request by chlunde**
AWS ALB Ingress Controller has been renamed to AWS Load Balancer Controller: https://aws.amazon.com/blogs/containers/introducing-aws-load-balancer-controller/

### CODE REVIEW

File: content/en/docs/concepts/services-networking/ingress-controllers.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Kubernetes as a project currently supports and maintains [GCE](https://git.k8s.i
   controller with [community](https://www.getambassador.io/docs) or 
   [commercial](https://www.getambassador.io/pro/) support from [Datawire](https://www.datawire.io/).
 * [AppsCode Inc.](https://appscode.com) offers support and maintenance for the most widely used [HAProxy](https://www.haproxy.org/) based ingress controller [Voyager](https://appscode.com/products/voyager). 
-* [AWS ALB Ingress Controller](https://github.com/kubernetes-sigs/aws-alb-ingress-controller) enables ingress using the [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/).
+* [AWS Load Balancer Controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller) (formerly known as AWS ALB Ingress Controller) enables ingress using the [AWS Application Load Balancer](https://aws.amazon.com/elasticloadbalancing/).
```
Comment (sftim) [inline content/en/docs/concepts/services-networking/ingress-controllers.md:30]:
```suggestion
* [AWS Load Balancer Controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller) (formerly known as AWS ALB Ingress Controller) enables ingress using the [AWS Elastic Load Balancing](https://aws.amazon.com/elasticloadbalancing/).
```

**Review by sftim — COMMENTED** [review]
/sig network
/kind cleanup

AWS Load Balancer Controller can also provision, manage and deprovision NLBs. I suggested a wording tweak.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-24777--kubernetes-io-master-staging.netlify.app/docs/concepts/services-networking/ingress-controllers/#additional-controllers

File: content/en/docs/concepts/services-networking/ingress-controllers.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/concepts/services-networking/ingress-controllers.md:30]: Hi @chlunde . Thanks for opening this pull request.
The changes look good.
What do you think about incorporating this suggestion?

**Comment by chlunde** [comment]
> AWS Load Balancer Controller can also provision, manage and deprovision NLBs. I suggested a wording tweak.

Excellent point! Sorry I missed the notification.

**Comment by kbhawkey** [comment]
Thanks @chlunde .
/lgtm

**Pull request merged**